src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double width;
+    private double length;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double width, double length, double height) {
+        super(color);
+        this.width = width;
+        this.length = length;
+        this.height = height;

```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.